### PR TITLE
HOFF-239: Update text on landing page

### PR DESCRIPTION
**Pull request by al01010**
What? 

Update text on the landing page, confirmation and the black banner. 

Why?  

Update text on the landing page - to give the forms more context. Text was also updated on the confirmation page on the banner to be more suitable for the demo.

### CODE REVIEW

File: apps/demo/translations/src/en/fields.json [hunk 1]
```diff
@@ -3,13 +3,16 @@
     "legend": "Which form would you like to explore?",
     "options": {
       "basic-form": {
-        "label": "Basic form"
+        "label": "Basic form",
+        "hint": "An example of a simple form produced using the hof framework. This form demos common components used by services using the hof framework."
```
Comment (MCrawleyHomeOffice) [inline apps/demo/translations/src/en/fields.json:7]: Could you uppercase the "HOF"? I'd also maybe put that it "showcases components and features" available in the HOF framework" instead.

File: apps/demo/translations/src/en/fields.json [hunk 2]
```diff
@@ -3,13 +3,16 @@
...
       },
       "complex-form": {
-        "label": "Complex form"
+        "label": "Complex form",
+        "hint": "An example of a form produced using the hof framework. This longer demo includes some steps from the basic form and features less common components used in other services using hof."
       },
```
Comment (MCrawleyHomeOffice) [inline apps/demo/translations/src/en/fields.json:12]: I'd maybe change this to say that it "showcases more advanced features available in the HOF framework" as well.

File: apps/demo/translations/src/en/fields.json [hunk 3]
```diff
@@ -3,13 +3,16 @@
...
       "build-your-own-form": {
-        "label": "Build your own form"
+        "label": "Build your own form",
+        "hint": "Link to the HOF documentation, with instructions on how to build your own hof form."
```
Comment (MCrawleyHomeOffice) [inline apps/demo/translations/src/en/fields.json:15]: Can we uppercase "HOF" please?

**Review by MCrawleyHomeOffice — CHANGES_REQUESTED** [review]

File: apps/demo/translations/src/en/fields.json (hunk 1 above)
Comment (al01010) [reply apps/demo/translations/src/en/fields.json]: changed hof to HOF and have altered the text.

File: apps/demo/translations/src/en/fields.json (hunk 2 above)
Comment (al01010) [reply apps/demo/translations/src/en/fields.json:11]: Made the suggested changes to the second sentence.

File: apps/demo/translations/src/en/fields.json (hunk 3 above)
Comment (al01010) [reply apps/demo/translations/src/en/fields.json]: Made this change

**Review by MCrawleyHomeOffice — APPROVED** [review]

**Pull request merged**